@xen-orchestra/fs/src/s3.js

```diff
@@ -456,7 +456,7 @@ export default class S3Handler extends RemoteHandlerAbstract {
         this.#s3.middlewareStack.use(getApplyMd5BodyChecksumPlugin(this.#s3.config))
       }
     } catch (error) {
-      if (error.Code !== 'ObjectLockConfigurationNotFoundError') {
+      if (error.Code !== 'ObjectLockConfigurationNotFoundError' && error.$metadata.httpStatusCode !== 501) {
```

Comment:
Shouldn't it be an *or*?

Reply:
My bad it's ok.

Feel free to reverse the condition if you wish :slightly_smiling_face: